content/sv/codebase.md

```diff
@@ -0,0 +1,17 @@
+## I. Codebase (Kodbas)
```

Comment:
The translations have either translated the heading completely or translated the heading and added ` (Codebase)` after.

```suggestion
## I. Kodbas
```

```suggestion
## I. Kodbas (Codebase)
```

This is the same for all 12 factors